bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/GeneratedInterop/CiphertextBallotSelection.generated.cs

```diff
@@ -287,7 +288,8 @@ CiphertextBallotSelectionHandle handle
                 SetLastError = true)]
             internal static extern Status GetProof(
                 CiphertextBallotSelectionHandle handle,
-                out NativeInterface.DisjunctiveChaumPedersenProof.DisjunctiveChaumPedersenProofHandle objectId
+                out NativeInterface.RangedChaumPedersenProof.RangedChaumPedersenProofHandle objectId
+                // out NativeInterface.DisjunctiveChaumPedersenProof.DisjunctiveChaumPedersenProofHandle objectId
```

Comment:
what do you want to do about the commented lines throughout?

Reply:
Removing them.  thx